theme/fonts/fonts.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
```

Comment:
Should use the same fonts as the main site.

Reply:
I don't really like font awesome 4.0. Where else should I put the SVG? I prefer keeping it in line because it allows me to use the current color attribute. I don't have to statically set the color.

Reply:
The original website font is just the system font. I chose Rubik because it looks nice. Could we consider changing the font? Of course, I'm fine changing it back.